test/e2e/scale.go

```diff
@@ -96,6 +108,15 @@ type scaleSpecInput struct {
 	// Note: Please note that the test suit will still fail since c6 creation failed. FailFast will determine
 	// if the test suit should fail as soon as c6 fails or if it should fail after all cluster creations are done.
 	FailFast bool
+
+	// SkipCleanup is set to true will skip deleting the workload clusters.
+	SkipCleanup bool
```

Comment:
I would suggest to set it like this in scale_test.go
```
			SkipCleanup:              skipCleanup,
```
This sets SkipCleanup to the value of the cmd flag like in other tests